website/src/config.ts

```diff
@@ -11,7 +11,7 @@ let _config: WebsiteConfig | null = null;
 let _runtimeConfig: RuntimeConfig | null = null;
 
 function getConfigDir(): string {
-    const configDir = import.meta.env.CONFIG_DIR;
+    const configDir = import.meta.env.PUBLIC_CONFIG_DIR;
```

Comment:
I saw some comment in your PR but could you add a summary of the reasoning behind this change in the main PR message (i.e. top post) - I don't know whether this env variable is used by other parts.

Is there something in the astro changelog?

Reply:
Reading more on this, and looking at where we specify `CONFIG_DIR` I'm pretty sure we should not do it this way. Can you revert and try to find an alternative solution that doesn't involve renaming the variable?

One definitely shouldn't rename a variable only partially. And in this case an env variable is part of the public API so that's two reasons not to change.

![Uploading Code 2024-12-05 14.12.59.png…]()

Reply:
We could use `process.env`. We don't need the variable at build time and we know that we're running on node (i.e. `process.env` is available).